dev-app/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "dev-app",
+  "private": true,
+  "scripts": {
+    "dev": "[[ -n $CI ]] && echo 'Cannot run this pnpm script on CI' && exit 1 || ibazel run //dev-app:build.serve",
+    "dev:build": "[[ -n $CI ]] && echo 'Cannot run this pnpm script on CI' && exit 1 || bazel build //dev-app:build"
```

We don't need to check for CI as we aren't going to accidentally deploy this to our docs site using this command. 

```suggestion
    "dev": "ibazel run //dev-app:build.serve",
    "dev:build": "bazel build //dev-app:build"
```